plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/operations/ExpandConstraintOperation.java

```diff
@@ -45,7 +45,7 @@
 	private final LinkedList<IGraphicalFeature> affectedFeatureList = new LinkedList<IGraphicalFeature>();
 
 	/**
-	 * @param featureModel
+	 * @param currentModel
 	 * @param label
```

Comment:
The whole comment block does not make sense without any data and this change introducing another wrong variable name does not help either.